src/enrich/lib/controllers/csv.js

```diff
@@ -148,18 +117,19 @@ const enrichTab = (data, response) => {
     el = Object.assign(el, res);
   });
 
-  return { lineEnriched, enrichedTab };
-};
+  return { enrichedArray, lineEnriched };
+}
 
 /**
- * write enriched date in enriched file
- * @param {array<object>} data array of line enriched
- * @param {String} headers headers
- * @param {char} separator separator of enriched file
- * @param {String} enrichedFile filepath of enriched file
- * @param {String} stateName - state filename
+ * Write enriched data in enriched file.
+ *
+ * @param {array<object>} data - Array of line enriched
+ * @param {String} headers - Headers
+ * @param {char} separator - Separator of enriched file
+ * @param {String} enrichedFile - Filepath of enriched file
+ * @param {String} stateName - State filename
```

Comment:
Do not confuse `String` with `string` (cf. https://www.geeksforgeeks.org/what-is-the-difference-between-string-and-string-in-typescript/)
```suggestion
 * @param {String} headers - Headers
 * @param {char} separator - Separator of enriched file
 * @param {String} enrichedFile - Filepath of enriched file
 * @param {String} stateName - State filename
```